clar2wasm/tests/wasm-generation/conditionals.rs

```diff
@@ -35,3 +33,70 @@ proptest! {
         )
     }
 }
+
+proptest! {
+    #[test]
+    fn match_optional_some(
+        (initial, some_val, none_val) in (prop_signature(), prop_signature())
+            .prop_flat_map(|(original_ty, ty)| {
+                (PropValue::from_type(original_ty), PropValue::from_type(ty.clone()), PropValue::from_type(ty))
+            })
+        )
+    {
+        crosscheck(
+            &format!(r#"(match (some {initial}) value {some_val} {none_val})"#),
```

Comment:
There's no reason to use r#-prefixed strings if you're not including literal quotation marks.

Reply:
It doesn't hurt either? But sure, I can remove it.

Reply:
done in d2a831b